### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ohadbitt**

### CODE REVIEW

**Review by AsafMah — APPROVED** [review]
Looks good!
Threw it in to gpt-4 and that's what it had to say:

In the "Bug report" template:

The subsection title:
Change ***Exception or Stack Trace*** to **Exception or Stack Trace** for consistency with other subsection titles (consider using two asterisks as in other subsections).
In the "Feature request" template:

In the sentence "Ex. I'm always frustrated when [...]" within the "Is your feature request related to a problem? Please describe." section:

Consider replacing "Ex." with "For example," or "e.g.," for clarity.
Consider adding a prompt for the user to add specific details to help flesh out the feature request, similar to the checklist in the bug report template. This would ensure the user provides all necessary details.

Add a period after the "Feature request" about description for consistency. The sentence should read as "Suggest an idea for this project."

Additionally, you can make the following enhancements for clarity and completeness:

In both templates, within the title fields (title: ''), you could provide an example or guidance on how to format the title for a bug report or feature request, if applicable.

For the bug report, under the "To Reproduce" subsection, you might want to include a line like "Steps to reproduce the behavior" followed by a numbered list where users can fill in the steps. This would guide the user in providing a thorough reproduction path.



Take of that what you want :)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+***Exception or Stack Trace***
+Add the exception log and stack trace if available
+
+**To Reproduce**
+The arguments used to reproduce the behavior
```
Comment (yihezkel) [inline .github/ISSUE_TEMPLATE/bug_report.md:17]: "arguments, steps, and/or code"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+***Exception or Stack Trace***
+Add the exception log and stack trace if available
```
Comment (yihezkel) [inline .github/ISSUE_TEMPLATE/bug_report.md:14]: "trace if" --> "trace, if"

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+The arguments used to reproduce the behavior
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Setup (please complete the following information):**
+ - .Net SDK Version: [e.g. .Net 6]
```
Comment (yihezkel) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: nit: Official way to write it is ".NET"

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
Comment (yihezkel) [inline .github/ISSUE_TEMPLATE/feature_request.md:11]: I would remove that example

**Review by yihezkel — APPROVED** [review]
The PR is primarily about text, so my comments are nits to improve the English and professionalism.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ohadbitt) [reply .github/ISSUE_TEMPLATE/bug_report.md:17]: Its just an exe tool - no steps or code here

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 4 above)
Comment (ohadbitt) [reply .github/ISSUE_TEMPLATE/feature_request.md:11]: This page section is just generated by github

**Pull request merged**